lib/passport/callbacks.coffee

```diff
@@ -130,7 +130,10 @@ onAccessToken = (req, done, params) -> (err, res) ->
   # /user API. Then attempt to fetch the access token again from Gravity and
   # recur back into this onAcccessToken callback.
   else if msg.match('no account linked')?
-    if (req?.session?.signupIntent && params?) then params.sign_up_intent = req.session.signupIntent
+    if (req?.session? && params?)
+      { sign_up_intent, receive_emails, accepted_terms_of_service } = req.session
+      params = { ...params, ...{ sign_up_intent, receive_emails, accepted_terms_of_service } }
```

Comment:
👍 little spread action

Reply:
I noticed this worked but does not work in force's coffeescript. or maybe that was coffeelint's force settings?

Reply:
You know, I was wondering actually if this would work. Mostly b/c, we use it all over in Metaphysics, Reaction, etc...but I can't recall seeing/using it in Force. 

I would not be surprised if the Coffeescript version in Force is one that doesn't include it?